Source/WebCore/html/HTMLElement.h

```diff
@@ -162,7 +162,7 @@ class HTMLElement : public StyledElement {
     void setPopover(const AtomString& value) { setAttributeWithoutSynchronization(HTMLNames::popoverAttr, value); };
     void popoverAttributeChanged(const AtomString& value);
 
-    virtual void handleInvokeInternal(const AtomString&) { }
+    virtual bool handleInvokeInternal(const HTMLFormControlElement* invoker, const AtomString&);
```

Comment:
The current change should be moved to the Element.h file.
The below change should be made here.

```suggestion
   bool handleInvokeInternal(const HTMLFormControlElement* invoker, const AtomString&) override;
```